core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -221,39 +208,19 @@
       + "this setting unique for each cluster.";
 
   protected static final String KAFKASTORE_CONNECTION_URL_DOC =
-      "Zookeeper URL for the Kafka cluster";
+      "ZooKeeper URL for the Kafka cluster";
```

Comment:
Is this still needed, along with `KAFKASTORE_CONNECTION_URL` ?

Reply:
It's still used by the SR ACL Authorizer security plugin.